pkg/tsdb/cloudwatch/mocks/oam_client.go

```diff
@@ -9,12 +11,12 @@ type FakeOAMClient struct {
 	mock.Mock
 }
 
-func (o *FakeOAMClient) ListSinks(input *oam.ListSinksInput) (*oam.ListSinksOutput, error) {
+func (o *FakeOAMClient) ListSinksWithContext(ctx aws.Context, input *oam.ListSinksInput, opts ...request.Option) (*oam.ListSinksOutput, error) {
```

Comment:
Doesn't the ctx need to be of type context.Context?

Reply:
Yes, it should be. I guess it got replaced due to my vscode env. Will fix and push a commit.